packages/core/src/fonts.css

```diff
@@ -0,0 +1,125 @@
+/* Fonts ......................................................... */
+
+/* Open Sans 300 .................................................... */
+
+/* woff2 - latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: normal;
+	font-weight: 300;
+	src: local('Open Sans Light'), local('OpenSans-Light'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.light.latin-ext.woff2)
+			format('woff2');
+}
+/* woff2 - latin */
+/* woff - latin + latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: normal;
+	font-weight: 300;
+	src: local('Open Sans Light'), local('OpenSans-Light'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.light.latin.woff2)
+			format('woff2'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.light.latin.latin-ext.woff)
+			format('woff');
+}
+
+/* Open Sans 400 .................................................... */
+
+/* woff2 - latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: normal;
+	font-weight: 400;
+	src: local('Open Sans Regular'), local('OpenSans-Regular'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin-ext.woff2)
+			format('woff2');
+}
+/* woff2 - latin */
+/* woff - latin + latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: normal;
+	font-weight: 400;
+	src: local('Open Sans Regular'), local('OpenSans-Regular'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.woff2)
+			format('woff2'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.regular.latin.latin-ext.woff)
+			format('woff');
+}
+
+/* Open Sans 400 italic ............................................. */
+
+/* woff2 - latin-ext */
+@font-face {
+	font-family: 'Open Sans';
+	font-style: italic;
+	font-weight: 400;
+	src: local('Open Sans Italic'), local('OpenSans-Italic'),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') URL for "woff" format is expected. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Afont-face-browser-compatibility)